app/services/attendance_csv_importer.rb

```diff
@@ -27,18 +27,17 @@ def process_attendances
     file_names = @client.list_file_names(@source_bucket)
     contents = file_names.map { |file_name| @client.get_file_contents(@source_bucket, file_name) }
     contents.each_with_index do |body, index|
-      parsed_csv = CsvParser.new(body).call
-      parsed_csv.each { |row| process_row(row, file_names[index]) }
+      @file_name = file_names[index]
+      @business = business
+      CsvParser.new(body).call.each { |row| process_row(row) }
```

Comment:
Deleted the parsed_csv assignment in order to satisfy rubocop's function line limit